docs/source/data/kedro_dataset_factories.md

```diff
@@ -1,7 +1,46 @@
 # Kedro dataset factories
 You can load multiple datasets with similar configuration using dataset factories, introduced in Kedro `0.18.12`.
 
-The syntax allows you to generalise your configuration and reduce the number of similar catalog entries by matching datasets used in your project's pipelines to dataset factory patterns.
+The dataset factories introduce a syntax that allows you to generalise your configuration and reduce the number of similar catalog entries by matching datasets used in your project's pipelines to dataset factory patterns.
+
+For example:
+```yaml
+factory_data:
+  type: pandas.CSVDataset
+  filepath: data/01_raw/factory_data.csv
+```
+
+With dataset factory, it can be re-written as:
+```yaml
+{placeholder}_data:
+  type: pandas.CSVDataset
+  filepath: data/01_raw/{placeholder}.csv
```

Comment:
```suggestion
  filepath: data/01_raw/{placeholder}_data.csv
```

Reply:
I agree it's quite similar. What I was trying to do is to separate this part at the top as a high level explanation. Since all other sub-section with (##) is more like a "how-to-do-x" so when I look at other sub-section I don't necessary check the other sub-sections.

Would it makes sense if I simply merge ##How to generalise dataset with similar name into the `#Kedro dataset factories` section?